charts/prometheus-pushgateway/Chart.yaml

```
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: "v1.11.0"
+appVersion: "v1.11.1"
```

You meant to bump `version`, did you not?

```suggestion
appVersion: "v1.11.0"
```